src/Contract/Monad.purs

```diff
@@ -121,7 +121,7 @@ import Wallet.Spec (WalletSpec)
 -- | All useful functions written in `QueryM` should be lifted into the
 -- | `Contract` monad and available in the same namespace. If anything is
 -- | missing, please contact us.
-newtype Contract (r :: Row Type) (a :: Type) = Contract (QueryMExtended r a)
+newtype Contract (r :: Row Type) (a :: Type) = Contract (QueryMExtended r Aff a)
```

Comment:
We need a Parallel instance for Contract too.

Reply:
I added the `Parallel` instance to `Contract` by creating a `ParContract` associated type. Are we interested in exporting this datatype? Currently I'm exporting the type (which should give some context to our users through the doc string, if they want to use the `Parallel` instance) but I'm not exporting the constructor.

Reply:
`parallel` is equivalent to the constructor anyway, so that's OK. But if you don't want to export the constructor, don't add the `Newtype` class instance.

Reply:
OK, I will remove the `Newtype` and leave it as that